schemas/+mice/+GUIs/modifyLine.m

```diff
@@ -30,7 +30,7 @@ function modifyLine(src,~)
 m.modify_box = get(h.modify_box,'value');
 
 if m.modify_box == 1
-    s = getEnumValues(mice.Lines.table,'line');
+    s = fetchn(mice.Lines,'line');
     s = [' ' s];
```

Comment:
This is probably a bug.  If s is a cell array of strings, then concatenating with a string won't work.  Has this line been tested?